kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/gaitup/PhysilogUploadProcessorFactory.kt

```diff
@@ -18,7 +18,7 @@ import java.time.format.DateTimeFormatter
 
 open class PhysilogUploadProcessorFactory(
         private val logRepository: LogRepository,
-        private val uploaderCreate: () -> FileUploaderFactory.FileUploader
+        private val uploaderCreate: FileUploaderFactory.FileUploader
```

Comment:
Apparently I wasn't completely clear: having this as a function is still good. Especially SFTP session may time out if it's a singleton. It's just that the function should not be called more than once per upload.

Reply:
I think I have already reverted that change in my last commit. Can you refresh?